assets/home/sections/quick-links.js

```diff
@@ -2,21 +2,135 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, external } from '@wordpress/icons';
+import { useState, useRef } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Section from '../section';
+import { Grid, Col } from '../grid';
+
+const quickLinksSpecialMapping = {
+	'sensei://install-demo-course': ( { setTitle, originalTitle, timer } ) => {
+		setTitle( 'Installing' );
+		const installedMessage = 'Installed!';
+		const clear = () => {
+			if ( timer.current ) {
+				clearTimeout( timer.current );
+			}
+			timer.current = 0;
+		};
+		const run = () => {
+			setTitle( ( oldTitle ) => {
+				if ( installedMessage === oldTitle ) {
+					clear();
+					return originalTitle;
+				}
+				if ( ! oldTitle.includes( '...' ) ) {
+					timer.current = setTimeout( run, 500 );
+					return oldTitle + '.';
+				}
+				timer.current = setTimeout( run, 2000 );
+				return installedMessage;
+			} );
+		};
+		clear();
+		timer.current = setTimeout( run, 500 );
+	},
+};
 
 /**
- * Quick Links section component.
+ * Component that shows a Quick Link.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.item The item to show.
+ */
+const QuickLink = ( { item } ) => {
+	let { url, title: originalTitle } = item;
+	const [ title, setTitle ] = useState( originalTitle );
+	const timer = useRef( 0 );
+
+	let onClick = null;
+
+	const callback = quickLinksSpecialMapping[ url ];
+	if ( !! callback ) {
+		url = '#';
+		onClick = ( e ) => {
+			e.preventDefault();
+			callback( { setTitle, originalTitle, timer } );
+		};
+	}
+	const isExternal = url && url !== '#' && ! url.includes( '/wp-admin/' );
```

Comment:
Maybe this check should be based on the domain? If we decide to add links to the frontend at some point, they would be considered external links.

Reply:
You mean checking if `isExternal` includes `senseilms.com? Or maybe if it's different than `window.location.host`?

Reply:
Take into account that I have made some changes [here](https://github.com/Automattic/sensei/pull/5952) to improve responsiveness of `<Link>` component in small screens.

Reply:
Hey, I decided to kind of rebase this over `add/sensei-guides-section` in [1393d34](https://github.com/Automattic/sensei/pull/5896/commits/1393d3417a35155bfe3f5e321bbb17b4e82378a0), and adapted this section to use the Link component here:

1. [d555050](https://github.com/Automattic/sensei/pull/5896/commits/d55505051c4dd8e7a44178e6a3af1401f224f72e);
2. [dbfe103](https://github.com/Automattic/sensei/pull/5896/commits/dbfe103f3129b7701f5b466eed7b5d479b69cae8)
3. [0c0b7fe](https://github.com/Automattic/sensei/pull/5896/commits/0c0b7fed21e3c6f274078a3bcd481c2b7f69cdb9)

Thanks!

Reply:
You can merge now!